### PR TITLE
test: migrated keep-alive-timeout.test.js from tap to node:test

### PR DESCRIPTION
**Pull request by Tony133**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

Proposal:

   - migrated `keepAliveTimeout.test.js` file from `tap` to `node:test` 🔥

### CODE REVIEW

File: test/keepAliveTimeout.test.js [hunk 1]
```diff
@@ -2,34 +2,33 @@
...
-  t.equal(httpServer.keepAliveTimeout, 1)
+  t.assert.strictEqual(httpServer.keepAliveTimeout, 1)
 
   const httpsServer = Fastify({ keepAliveTimeout: 2, https: {} }).server
-  t.equal(httpsServer.keepAliveTimeout, 2)
+  t.assert.strictEqual(httpsServer.keepAliveTimeout, 2)
 
   const http2Server = Fastify({ keepAliveTimeout: 3, http2: true }).server
-  t.not(http2Server.keepAliveTimeout, 3)
+  t.assert.notEqual(http2Server.keepAliveTimeout, 3)
```
Comment (climba03003) [inline test/keepAliveTimeout.test.js:31]:
```suggestion
  t.assert.notStrictEqual(http2Server.keepAliveTimeout, 3)
```

**Comment by Uzlopak** [comment]
Please rename file to kebab-case.

**Comment by Tony133** [comment]
> Please rename file to kebab-case.

Done! see here: https://github.com/fastify/fastify/pull/5796/files

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**